delfin/drivers/dell_emc/vmax/client.py

```diff
@@ -401,6 +401,32 @@ def list_ports(self, storage_id):
 
             return port_list
 
+    def list_disks(self, storage_id):
+        if int(self.uni_version) < 91:
+            return []
+        try:
+            # Get list of Disks
+            disks = self.rest.get_disk_list(self.array_id,
+                                            self.uni_version)
+            disk_list = []
+            for disk in disks:
+                disk_info = self.rest.get_disk(
+                    self.array_id, self.uni_version, disk)
+
+                disk_item = {
+                    'name': 'disk_' + disk,
```

Comment:
Do we need to prefix 'disk_' , if we do so it will be different from the element manager showed disk name!